services/invitations/src/simcore_service_invitations/api/_invitations.py

```diff
@@ -0,0 +1,123 @@
+import logging
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from pydantic import BaseModel, Field, HttpUrl
+
+from ..core.settings import WebApplicationSettings
+from ..invitations import (
+    InvalidInvitationCode,
+    InvitationContent,
+    InvitationInputs,
+    create_invitation_link,
+    extract_invitation_content,
+    parse_invitation_code,
+)
+from ._dependencies import get_current_username, get_settings
+
+logger = logging.getLogger(__name__)
+
+
+#
+# API SCHEMA MODELS
+#
+
+INVALID_INVITATION_URL_MSG = "Invalid invitation link"
+
+
+class InvitationCreate(InvitationInputs):
+    class Config:
+        # Same as InvitationInputs but WITHOUT alias
+        fields = {
+            "issuer": {
+                "alias": None,
+            },
+            "guest": {
+                "alias": None,
+            },
+            "trial_account_days": {
+                "alias": None,
+            },
+        }
+        schema_extra = {
+            "example": {
+                "issuer": "issuerid",
+                "guest": "invitedguest@company.com",
+                "trial_account_days": None,
```

Comment:
Yes - I do have a github account :) So as requested i also add my comment here:
This field is currently not present in the `openapi.json` file - maybe a question just running some magic `regenerate schema` command?

Reply:
now it should be fine. There was an issue with alias names